tinyxml2.cpp

```diff
@@ -887,6 +887,17 @@ char* XMLNode::ParseDeep( char* p, StrPair* parentEnd )
             break;
         }
 
+        XMLElement* ele = node->ToDeclaration();
+        if ( decl ) {

```

Comment:
Doh! :-/

I should really get around to using git, rather than editing in github.